### PR TITLE
Add 2.1.1 docs

### PR DESCRIPTION
**Pull request by ruanwenjun**

### CODE REVIEW

**Review by wuchunfu — CHANGES_REQUESTED** [review]
It is recommended to change 
```
{
    label: versions[0],
    to: `docs/${versions[0]}/introduction`,
},
```
 to 
```
{
    label: versions[0],
    to: `docs/${versions[0]}/intro/about`,
},
```
 in the `docusaurus.config.js` file

File: src/pages/versions/config.json [hunk 1]
```diff
@@ -50,10 +50,10 @@
       "nextLink": "/docs/intro/about",
       "latestData": [
         {
-          "versionLabel": "2.1.0",
-          "docUrl": "/docs/2.1.0/introduction",
-          "downloadUrl": "https://github.com/apache/incubator-seatunnel/releases/tag/2.1.0",
-          "sourceTag": "2.1.0"
+          "versionLabel": "2.1.1",
+          "docUrl": "/docs/2.1.1/introduction",
```
Comment (wuchunfu) [inline src/pages/versions/config.json:54]: Change `"docUrl": "/docs/2.1.1/introduction",` to `"docUrl": "/docs/2.1.1/intro/about",`

File: src/pages/versions/config.json [hunk 2]
```diff
@@ -63,6 +63,12 @@
         }
       ],
       "historyData": [
+        {
+          "versionLabel": "2.1.1",
+          "docUrl": "/docs/2.1.1/introduction",
```
Comment (wuchunfu) [inline src/pages/versions/config.json:68]: Change `"docUrl": "/docs/2.1.1/introduction",` to `"docUrl": "/docs/2.1.1/intro/about",`

**Comment by ruanwenjun** [comment]
@wuchunfu Thanks for your review, done.

File: README.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ asf-staging #The asf-staging official website test environment is accessed throu
 
 This website is compiled using node, using Docusaurus framework components
 
-1. Download and install nodejs (version>12.5.0)
+1. Download and install nodejs (version>14)
```
Comment (zhongjiajie) [inline README.md:24]: good catch!

File: docusaurus.config.js [hunk 4]
```diff
@@ -85,7 +85,8 @@ const config = {
                     items: [
                         {
                             label: versions[0],
-                            to: `docs/${versions[0]}/introduction`,
+                            to: `docs/${versions[0]}/intro/about`,
+
                         },
                         ...versions.slice(1).map((version) => ({
```
Comment (zhongjiajie) [inline docusaurus.config.js:91]: NIT: did we want to show all history docs in our website?

Comment (ruanwenjun) [reply docusaurus.config.js:91]: We can add a limit if the history is long.

**Review by zhongjiajie — APPROVED** [review]
LGTM,  @wuchunfu could you please take a look one more time?

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**